plugin/dnstap/const.go

```diff
@@ -0,0 +1,16 @@
+package dnstap
+
+// Mirror some of the tap. constants so we can reference them from this plugin
+// and not the vendored directory meaning out-of-tree middleware can use this.
+
+import (
+	tap "github.com/dnstap/golang-dnstap"
+)
+
+// Mirror constants from the golang-dnstap package.
+const (
+	SocketProtocolTCP        = tap.SocketProtocol_TCP
```

Comment:
Would be best placed in dnstap/msg because those are values for field Type of payload type tap.Message, only a detail though.

Reply:
all of them? sgtm